lib/project_types/theme/messages/messages.rb

```diff
@@ -2,12 +2,47 @@
 module Theme
   module Messages
     MESSAGES = {
-      ensure_themekit_installed: {
-        downloading: "Downloading Themekit %s",
-        failed: "Download failed",
-        successful: "Themekit installed successfully",
-        unsuccessful: "Unable to verify download digest",
-        verifying: "Verifying download...",
+      theme: {
+        create: {
+          creating_theme: "Creating theme %s",
+          checking_themekit: "Verifying Theme Kit",
+          failed: "Theme could not be created",
```

Comment:
```suggestion
          failed: "Couldn’t create the theme",
```

Shopify uses contractions in UI copy, and we prefer active to passive voice.

Is it possible to be any more specific about what's failed? Would it differentiate between, say, a file-permission error vs a network error?

Reply:
The output from Theme Kit is displayed within the frame, which would consist of more specific errors. For example:

![image](https://user-images.githubusercontent.com/54586536/89832780-d1e9e780-db2d-11ea-882c-63eeb9c14522.png)

Reply:
Ah, I see. Then all good with just the reword! 👍🏻